travis-build.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+set -e
+
+find . -name '*.pyc' -delete
+
+envsubst '$PY_VERSION' <Dockerfile.tmpl > Dockerfile
+
+echo 'building pyinstaller image'
+docker build --pull -t pyinstaller .
+
+echo 'creating pyinstaller container'
+2>/dev/null docker rm -v pyinstaller || true
```

Comment:
Output redirects should go *behind* the command, since the *command* is the important think (reading left-to-right).